docs/spec/auth/scope.md

```diff
@@ -0,0 +1,133 @@
+<!--[metadata]>
++++
+title = "Token Scope Documentation"
+description = "Describes the scope and access fields used for registry authorization tokens"
+keywords = ["registry, on-prem, images, tags, repository, distribution, advanced, access, scope"]
+[menu.main]
+parent="smn_registry_ref"
++++
+<![end-metadata]-->
+
+# Docker Registry Token Scope and Access
+
+Tokens used by the registry are always restricted what resources they may
+be used to access, where those resources may be accessed, and what actions
+may be done on those resources. Tokens always have the context of a user which
+the token was originally created for. This document describes how these
+restrictions are represented and enforced by the authorization server and
+resource providers.
+
+## Scope Components
+
+### Subject (Authenticated User)
+
+The subject represents the user for which a token is valid. Any actions
+performed using an access token should be considered on behalf of the subject.
+This is included in the `sub` field of access token JWT. A refresh token should
```

Comment:
A refresh token should be limited to a single subject **and** only be able to give out access tokens for that subject.